uiplib/gui/settingsTab.py

```diff
@@ -118,13 +132,25 @@ def file_open_helper(appObj):
     appObj.root.update_idletasks()
 
 
+def file_open_helper1(appObj):
+    """Open the dialog to choose where pictures are to be saved."""
+    directory1 = filedialog.askdirectory()
+    appObj.fav_pics_folder.set(directory1)
+    appObj.root.update_idletasks()
+
+
 def handle_settings(appObj):
     """Handler to update the settings file."""
     try:
         appObj.new_settings['pics-folder'] = appObj.pics_folder.get()
     except AttributeError as e:
         messagebox.showwarning("AttributeError", "No pics folder selected")
```

Comment:
Here instead of 2 `try`s have one try with both of them.
And you can warn with `Folder not selected`